harness/tests/experiment/pytorch/test_pytorch_trial.py

```diff
@@ -182,33 +185,33 @@ def make_workloads_1() -> workload.Stream:
             latest_checkpoint = interceptor.metrics_result()["uuid"]
             steps_completed = trainer.get_steps_completed()
 
-        controller1 = utils.make_trial_controller_from_trial_implementation(
+        controller_1 = utils.make_trial_controller_from_trial_implementation(
```

Comment:
nitpick: not a huge fan of using numbers in variable names. i know you didn't make this change, it was already there, but maybe we can change the name to something more descriptive, like invalid_controller, invalid_, etc?

Reply:
I totally agree with this nitpick. Done!